<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulador de colas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_arrival_rate">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Tasa de arribos:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>70</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_service_rate">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_num_servers">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>91</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_simulate">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Simular</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_num_servers">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>NÃºmero de servidores:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_service_rate">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Tasa de servicio:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_arrival_rate">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="btnMenu_archivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionSalir"/>
   </widget>
   <addaction name="btnMenu_archivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>lineEdit_arrival_rate</tabstop>
  <tabstop>lineEdit_service_rate</tabstop>
  <tabstop>lineEdit_num_servers</tabstop>
  <tabstop>btn_simulate</tabstop>
  <tabstop>btn_clear</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btn_clear</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_service_rate</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>475</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_clear</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_arrival_rate</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>473</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_clear</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_num_servers</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>522</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
